### ISSUE TITLE
problem transfering save files

### ISSUE THREAD

**Issue by DrWolfnstine** [hrydgard/ppsspp#8617]
Hello I am new to this so please don't be too harsh if this issue has been adressed already, I did a few searches but didn't find anything that exactly matched my issue. I am attempting to transfer a save file from ppsspp to my psp. The game I am playing is FFT WOTL. When I transfer FROM my psp to ppsspp it works just fine, I just copy the SAVEDATA folder from my psp and paste it into ppsspp's memstick/psp folder, and bam it loads up just fine. The problem comes when I attempt to do the reverse and copy the SAVEDATA folder from ppsspp to my psp. I have tried deleting the save data folder in my psp first, and I even copied the entire psp folder from ppsspp to my psp and overwrote everything in an attempt to get it work. The SAVEDATA folder copies and overwrites just fine, but when I load the actual game in my psp my saves are back to were they were BEFORE I started playing in ppsspp and saved farther into the game.
In other words all the progress I made in ppsspp is reverted to when I originally copied it from my psp to ppsspp in the first place. Any help would be appresiated, My ppsspp version is 1.2.1

**Comment by unknownbrackets** [comment]
That's strange.  If the PSP was rejecting those saves, I would expect an error, rather than it somehow reverting.

Have you checked the modification time on the files you're copying?  Maybe when you saved in PPSSPP, it put the saves somewhere else.  Maybe the files you are copying aren't the right ones?

One way to check this would be to rename "SAVEDATA" to "SAVEDATA_TEST" on the device you have PPSSPP on.  Then try running PPSSPP - it should save there's no save, right?

If it can still load the save, that means you're definitely looking in the wrong place.

If you're on Windows, the normal place for saves is inside memstick/PSP/SAVEDATA.  However, you can configure it to store the saves in other places - like under My Documents/PPSSPP.

-[Unknown]

**Comment by DrWolfnstine** [comment]
Indeed it IS strange, which is why I am having problems understanding why my saves keep reverting, I tried what you suggested and renamed the SAVEDATA folder I was using to SAVEDATA_test and when I loaded up ppsspp it found no save data and made a new SAVEDATA folder next the now named SAVEDATA_test. So I am getting my saves from the correct place. Can't for the life of me figgure out what is causing my FFT saves to revert back to where they were before I starting playing on ppsspp even though I overwrote the saves on ppsspp.

**Comment by unknownbrackets** [comment]
Okay, then, next do the same thing on the PSP - blank out the PSP/SAVEDATA folder that you're copying to, and don't put anything in it.  Does the PSP then show no saves?

-[Unknown]

**Comment by DrWolfnstine** [comment]
Confirmed once deleted psp shows no saves.

**Comment by unknownbrackets** [comment]
Alright, can you zip up your save data somewhere and upload it?  This probably means we're saving in some wrong way that's not overwriting the old savedata, but is loading correctly either way from within PPSSPP.

-[Unknown]

**Comment by DrWolfnstine** [comment]
Done, link is below:

http://www.mediafire.com/download/aow5cpvpl449iap/SAVEDATA.zip

**Comment by DrWolfnstine** [comment]
For good mesure here is my entire memstick folder from ppsspp:

http://www.mediafire.com/download/u57rto1tjq31cx2/memstick.zip

**Comment by hrydgard** [comment]
Just got a report over email that the opposite isn't working - the user was playing on the PSP, transferred the saves to PPSSPP and the saves were not even visible. So seems like something is odd with this game..

**Comment by unknownbrackets** [comment]
We did make some saving changes since v1.6.3 iirc, so it's possible something was fixed?

I think I wasn't able to repro against a US region UMD.

-[Unknown]

**Comment by benderscruffy** [comment]
No feedback/outdated?
